api/resource/admin.py

```diff
@@ -137,26 +134,14 @@ def admin_update(email: str) -> Tuple[Dict[str, str], int]:
         if isinstance(body, str):
             return {"msg": body}, 400
 
-        admin = db.admins.find_one({"email": body["email"]})
-
-        if "password" in body:
-            if body["password"] != body["password_confirmation"]:
-                return (
-                    {"msg": "`password` and `password_confirmation` don't match"},
-                    401,
-                )
-            del body["password_confirmation"]
-            body["password"] = generate_password_hash(body["password"])
-        else:
-            body["password"] = admin["password"]
```

Comment:
This break the account recovery feature, why remove this?

Reply:
Gotch I didn't see that, reverting